pysatMadrigal/instruments/gnss_tec.py

```diff
@@ -290,3 +283,55 @@ def load(fnames, tag='', inst_id=''):
                      meta.labels.max_val: min_lon + 360.0}
 
     return data, meta
+
+
+def list_remote_files(tag, inst_id, start=dt.datetime(1900, 1, 1),
```

Comment:
```suggestion
def list_remote_files(tag, inst_id, start=dt.datetime(1998, 10, 15),
```

This is the earliest GNSS data entry in the database.